build.sbt

```
@@ -15,6 +15,11 @@ lazy val root = (project withId "stryker4s" in file("."))
     addCommandAlias(
       "publishM2Local",
       "set ThisBuild / version := \"SET-BY-SBT-SNAPSHOT\"; stryker4s-core/publishM2; stryker4s-api/publishM2"
+    ),
+    // Publish to .ivy folder for local testing
+    addCommandAlias(
+      "publishLocal",
```

```suggestion
      "publishCommandRunnerLocal",
```

publishLocal is an existing sbt task, so I'd like to keep the distinction of what this publishes

---

Thanks. I think I got it finally. We need a special publish command for command-runner, but it also needs to publish core and api as those are dependencies.